### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [privacyIDEA/privacyIDEA WebUI](https://hosted.weblate.org/projects/privacyidea/privacyidea-webui/).

Current translation status:

Weblate translation status

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -1048,8 +1053,8 @@ msgid ""
 "Details for machine {{ machineid }} in Machine Resolver\n"
 "        {{ machineresolver }}"
 msgstr ""
-"Details zur Maschine {{ machineid }} im Maschinen-Resolver "
-"{{ machineresolver }}"
+"Details zur Maschine {{ machineid }} im Maschinen-Resolver {{ "
+"machineresolver }}"
```
Comment (cornelinux) [inline po/de.po:1057]: This looks like an evil linebreak in the template. I am not sure, if this will work out!

File: po/de.po [hunk 2]
```diff
@@ -1088,8 +1093,8 @@ msgid ""
 "Do you really want to delete the Token {{ tokenSerial }} from user {{ token."
 "username }}?"
 msgstr ""
-"Wollen Sie wirklich den Token {{ tokenSerial }} des Benutzers {{ token."
-"username }} löschen?"
+"Wollen Sie wirklich den Token {{ tokenSerial }} des Benutzers {{ "
+"token.username }} löschen?"
```
Comment (cornelinux) [inline po/de.po:1097]: Evil linebreak.

File: po/de.po [hunk 3]
```diff
@@ -3964,8 +3967,8 @@ msgid ""
 "The password that needs to be combined with the OTP PIN:\n"
 "        <b>{{ lostResult.password }}</b>"
 msgstr ""
-"Das Passwort, das mit der alten OTP-PIN kombiniert werden muss: "
-"<b>{{ lostResult.password }}</b>"
+"Das Passwort, das mit der alten OTP-PIN kombiniert werden muss: <b>{{ "
+"lostResult.password }}</b>"
```
Comment (cornelinux) [inline po/de.po:3971]: Bad linebreak!?

Comment (plettich) [reply po/de.po:3971]: Luckily HTML ignores additional spaces and line-breaks:
grafik

**Review by plettich — APPROVED** [review]

**Review by plettich — APPROVED** [review]

**Pull request merged**